pkg/placement/placement-resolver.go

```diff
@@ -217,6 +217,7 @@ func (resolver *placementResolver) createResolution(placementKey string) *placem
 	placementResolution := &placementResolution{
 		objectIdentifierToKey: make(map[string]*util.Key),
 		destinations:          sets.New[string](),
+		workloadGeneration:    1,
```

Comment:
What if this process, or the node running it, crashes and later there is a replacement process?

Reply:
Indeed, for now keeping it like this for correctness under the assumption of friendly environment friendly user. Later on this either has to be synced on load, or this field has to be reevaluated.

Reply:
I do not know what you mean by "load". If you want to defer this then please open an Issue for it and update #1692

Reply:
done #1708